static/css/place/place.scss

```diff
@@ -171,6 +171,7 @@ $sidebar-top-padding: 132px;  // Keep in sync with place.ts: Y_SCROLL_LIMIT
 
 .chart-container h4 .sub-title {
   color: #999;
+  padding-left: 2px;
```

Comment:
Can we use .25rem instead (that way the space will scale with the base font size)